composer/workflows/kubernetes_pod_operator_c2.py

```diff
@@ -64,11 +64,10 @@
     start_date=YESTERDAY,
 ) as dag:
     # Only name, namespace, image, and task_id are required to create a
-    # KubernetesPodOperator. In Cloud Composer, currently the operator defaults
-    # to using the config file found at `/home/airflow/composer_kube_config if
-    # no `config_file` parameter is specified. By default it will contain the
-    # credentials for Cloud Composer's Google Kubernetes Engine cluster that is
-    # created upon environment creation.
+    # KubernetesPodOperator. In Cloud Composer, the config file found at
+    # `/home/airflow/composer_kube_config contains credentials for
```

Comment:
Nit: missing ` in the end o /home/airflow/composer_kube_config

Reply:
Thanks! Fixed.